docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -232,6 +232,10 @@ like so::
 
    $ pip install -r tests/requirements/py3.txt  # Python 2: py2.txt
 
+If you are experiencing any problems during the installation of these 
+dependencies, please see our :ref:`troubleshooting <troubleshooting-unit-testing>` 
+page.
```

Comment:
I don't think it makes a lot of sense that the general guide on how to address dependency issues should live under troubleshooting unit testing. Most people installing something like `argon2-cffi` are more likely than not end users and aren't running Django's unit tests.